### PR TITLE
Add target hse u575xg

### PR DESCRIPTION
**Pull request by lefebvresam**
### Summary of changes 

MCU family MCU_STM32U575xG:
- ROM and RAM size
- Added support for HSE and LSE MSI auto calibration
- Added USB clock config

### Documentation 

Has been discussed in mbed ce tickets.

----------------------------------------------------------------------------------------------------------------
### Pull request type 

    [] Patch update (Bug fix / Target update / Docs update / Test update / Refactor)
    [x] Feature update (New feature / Functionality change / New API)
    [] Major update (Breaking change E.g. Return code change / API behaviour change)

----------------------------------------------------------------------------------------------------------------
### Test results 

    [] No Tests required for this change (E.g docs only update)
    [] Covered by existing mbed-os tests (Greentea or Unittest)
    [x] Tests / results supplied as part of this PR
    
Tests with USB and HSI not done.
Tests are done with HSE and LSE.
    
----------------------------------------------------------------------------------------------------------------
### Reviewers 

----------------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: targets/TARGET_STM/TARGET_STM32U5/TARGET_STM32U575xG/cmsis_nvic.h [hunk 1]
```diff
@@ -22,15 +22,15 @@
 #endif
 
 #if !defined(MBED_ROM_SIZE)
-#define MBED_ROM_SIZE  0x0  // 0 B
+#define MBED_ROM_SIZE  0x100000 // 1GB
```
Comment (JohnK1987) [inline targets/TARGET_STM/TARGET_STM32U5/TARGET_STM32U575xG/cmsis_nvic.h:25]: A typo 1G instead of 1M probably?

**Comment by lefebvresam** [comment]
IndeedVerzonden vanaf mijn Galaxy
-------- Oorspronkelijk bericht --------Van: JohnK1987 ***@***.***> Datum: 20/01/2024  07:38  (GMT+01:00) Aan: mbed-ce/mbed-os ***@***.***> Cc: Sam Lefebvre ***@***.***>, Author ***@***.***> Onderwerp: Re: [mbed-ce/mbed-os] Add target hse u575xg (PR #218) 
@JohnK1987 commented on this pull request.



In targets/TARGET_STM/TARGET_STM32U5/TARGET_STM32U575xG/cmsis_nvic.h:
> @@ -22,15 +22,15 @@
 #endif
 
 #if !defined(MBED_ROM_SIZE)
-#define MBED_ROM_SIZE  0x0  // 0 B
+#define MBED_ROM_SIZE  0x100000 // 1GB

A typo 1G instead of 1M probably?

**Comment by JohnK1987** [comment]
I understand the code but the comment is wrong. The ROM definitely does not have 1 Giga bytes of size.
So the line should look like 
```
#define MBED_ROM_SIZE  0x100000 // 1MB
```

**Comment by lefebvresam** [comment]
Should be solved now

**Review by JohnK1987 — APPROVED** [review]

**Comment by lefebvresam** [comment]
Merging is blocked
The base branch restricts merging to authorized users. [Learn more about protected branches.](https://docs.github.com/articles/about-protected-branches/)

**Comment by JohnK1987** [comment]
Yeah, @multiplemonomials will do that.

**Comment by multiplemonomials** [comment]
Try it now, I think I fixed the permissions

**Comment by multiplemonomials** [comment]
And, thank you for your contribution!  I really appreciate it!

**Pull request merged**